src/components/Auth/SignUp/SignUp.react.js

```diff
@@ -259,33 +259,36 @@ class SignUp extends Component {
   };
 
   render() {
-    const {
-      email,
-      emailErrorMessage,
-      password,
-      passwordErrorMessage,
-      passwordScore,
-      passwordStrength,
-      confirmPassword,
-      passwordConfirmErrorMessage,
-      isCaptchaVerified,
-      captchaVerifyErrorMessage,
-      signupErrorMessage,
-      loading,
-      success,
-    } = this.state;
-    const { openSignUp, onRequestOpenLogin, captchaKey } = this.props;
+    const styles = {
+      width: '100%',
+      textAlign: 'center',
+      padding: '10px',
+    };
 
-    const isValid =
-      email &&
-      !emailErrorMessage &&
-      password &&
-      !passwordErrorMessage &&
-      confirmPassword &&
-      !passwordConfirmErrorMessage &&
-      isCaptchaVerified;
+    const fieldStyle = {
```

Comment:
We have created an explicit style object right at the top of the component code. Please place these object back there and destructure it in the render funciton.